components/massmint/Massmint.vue

```diff
@@ -70,6 +70,9 @@
         </template>
       </NeoDropdown>
     </section>
+    <section class="collapse-container">
```

Comment:
this class should be inside NeoCollapse componenet

Reply:
uh isnt the 40px margin specific to this page though?

Reply:
but the entire css in ```NeoCollapse.scss``` is dependent on being wrapped in this class

Reply:
yeah didnt notice that, fixed that